polaris.shopify.com/content/foundations/content/naming.md

```diff
@@ -693,8 +693,6 @@ confusing.
 
 Before naming a new product or feature:
 
-- Conduct an audit of our [vocabulary list](/content/vocabulary#navigation) to
```

Comment:
Could we replace what's being deleted here with:
- Conduct an audit of existing names to narrow down your naming choices